tests/e2e/scale/noobaa/test_hsbench.py

```diff
@@ -57,3 +75,28 @@ def test_s3_benchmark_hsbench(self, hsbenchs3):
 
         # Check ceph health status
         utils.ceph_health_check()
+
+    @bugzilla("1998680")
+    @skipif_ocs_version("<4.9")
+    @pytest.mark.polarion_id("OCS-2698")
+    def test_s3_benchmark_object_bucket(self, s3bench, mcg_obj, bucket_factory):
+        """
+        Tests large number of objects on a single bucket:
+        * Create an OBC
+        * Run hs S3 benchmark to create 700k objects on the object bucket
+        * Post writing objects verify OBC creation
+        """
+        # Create an Object bucket
+        object_bucket = bucket_factory(amount=1, interface="OC")[0]
+
+        # Write 1M objects to the object bucket
```

Comment:
This test is to write 70000 objects, not 1M.

Reply:
ack